schema.graphql

```diff
@@ -250,6 +250,22 @@ type DailyUniqueAddressInteraction @entity {
   addresses: [Bytes!]!
 }
 
+# Weekly unique users interactions
+type WeeklyUniqueAddressInteraction @entity {
+  id: ID! # timestamp formatted as YYYY-MM-DD date string
```

Comment:
This to clear up the confusion. The comment should: YYYY-WeekNumber. When running:

```
{
  weeklyUniqueAddressInteractions (orderBy: 
    id, orderDirection: desc
  ) {
    id 
    weekStart
    weekEnd
  }
}
```

I get:

```
{
  "data": {
    "weeklyUniqueAddressInteractions": [
      {
        "id": "2022-9",
        "weekStart": "2022-02-28",
        "weekEnd": "2022-03-06"
      },
      {
        "id": "2022-8",
        "weekStart": "2022-02-21",
        "weekEnd": "2022-02-27"
      },
      {
        "id": "2022-7",
        "weekStart": "2022-02-14",
        "weekEnd": "2022-02-20"
      },
      {
        "id": "2022-6",
        "weekStart": "2022-02-07",
        "weekEnd": "2022-02-13"
      },
      {
        "id": "2022-5",
        "weekStart": "2022-01-31",
        "weekEnd": "2022-02-06"
      },
      {
        "id": "2022-4",
        "weekStart": "2022-01-24",
        "weekEnd": "2022-01-30"
      },
 /// ....
      ````